doc/cookbook/index.md

```diff
@@ -156,3 +156,16 @@ For a full recipe on creating your own readmapper, see the very end of the tutor
 #  Serialise data structures with cereal
 
 \include doc/howto/use_cereal/load.hpp
+
+\section cookbook_custom_dna4_alphabet A custom dna4 alphabet that converts all unknown characters to `A`
+
+When assigning from `char` or converting from a larger nucleotide alphabet to a smaller one, *loss of information*
+can occur since obviously some bases are not available. When converting to seqan3::dna5 or seqan3::rna5,
+non-canonical bases (letters other than A, C, G, T, U) are converted to `'N'` to preserve ambiguity at that position.
```

Comment:
Did you check how <code>\`'N'`</code> looks like? I remember vaguely that this is sometimes not parsed correctly and <code>\<code>\</code> </code> needs to be used?

Reply:
It looks fine on firefox.

![image](https://user-images.githubusercontent.com/7955878/84229515-10413700-aaea-11ea-9e09-75e3ab963830.png)